src/Components/test/E2ETest/Infrastructure/BasicTestAppTestBase.cs

```diff
@@ -1,25 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Removing this makes a bunch of other tests seem more complicated (because they have to inherit from `ServerTestBase<ToggleExecutionModeServerFixture<Program>>`). What's the advantage of removing it?

Reply:
That it gets miss used. We were inheriting from BasicTestAppTestBase on a bunch of places (and getting a `ToggleExecutionModeServerFixture<Program>` in a bunch of places for server-only tests and that made it hard to determine what tests run where.

This way is one less concept and you are forced to choose the right fixture, which makes it clear what the scenario for the test is.

Reply:
OK that's fine. Thanks for explaining.